packages/react-component-library/src/components/DatePicker/DatePicker.tsx

```diff
@@ -339,45 +340,56 @@ export const DatePicker = ({
         aria-live="polite"
       >
         <FocusTrap focusTrapOptions={focusTrapOptions}>
-          <StyledDayPicker
-            firstDayOfWeek={1}
-            todayButton="Jump to today"
-            onTodayButtonClick={(day) => {
-              if (isDateDisabled(day, disabledDays)) {
-                return
-              }
+          <div>
+            <StyledJumpToToday
```

Comment:
I think ultimately a lot of this needs separating out as there is quite a bit going on in `DatePicker.tsx`.

Reply:
Ok cool I agree - we'll get the two that are in the queue through then I can open a separate PR for that.